pkg/system/syscall_windows.go

```diff
@@ -86,18 +82,9 @@ type osVersionInfoEx struct {
 
 // GetOSVersion gets the operating system version on Windows. Note that
 // docker.exe must be manifested to get the correct version information.
+// Deprecated: use github.com/Microsoft/hcsshim/osversion.Get() instead
 func GetOSVersion() OSVersion {
```

Comment:
Wondering if this function should be deprecated entirely, and have callers use the underlying source of truth in HCSShim?

Reply:
I plan removing it after some time; deprecating it first helps migration possible downstream users without breaking them immediately. Most editors mark functions that have a `Deprecated` comment, so this should put some attention to it.